metrics/prometheus.go

```diff
@@ -1155,8 +1155,19 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 		values := make([]string, 0, len(rawLabels))
 		labels := make([]string, 0, len(rawLabels))
 		containerLabels := c.containerLabelsFunc(cont)
+		duplicate := false
```

Comment:
maybe just declare `duplicate` inside the `for l := range rawLabels` block?  Then you don't need to reset it to false each loop.

Reply:
Yeah, confused myself by doing this a different way before submitting the PR. Thanks :)